### ISSUE TITLE
Naming concention enum pascal case does not detect single words.

### ISSUE THREAD

**Issue by Domvel** [ajafff/tslint-consistent-codestyle#128]
The naming-convention rule for enums in PascalCase does not detect single UPPER_CASE words.

Demo:

```ts
enum Foo {
  THIS_IS_DETECTED = 1,
  BUTTHISNOT = 2
}
```

`THIS_IS_DETECTED` is detected by this rule and underlined in vs-code.
`BUTTHISNOT` is not detected and tolerated / ignored.

tslint.json
```json
"naming-convention": [true, { "type": "enumMember", "format": "PascalCase" }]
```

Expected: 
Strict PascalCase. Which means alswys the first letter of a word is in upper case, the rest in lower case. Also acronyms, etc. No exceptions.

Examples:

| From Upper | To Pascal |
| --- | --- |
| LED | Led |
| ID | Id |
| UUID | Uuid |

version: 1.15.1

**Comment by ajafff** [comment]
You want to use `StrictPascalCase` instead of `PascalCase` as that checks for at least one lowercase character between uppercase characters
